src/bento/dashboardTabData.js

```diff
@@ -1022,3 +1022,34 @@ query fileOverview($file_uuids: [String], $offset: Int = 0, $first: Int = 100000
   }
 }
   `;
+
+export const GET_FILE_IDS_FROM_FILE_NAME = gql`
+query (
+    $file_name: [String],
+    $offset: Int,
+    $first: Int,
+    $order_by: String
+)
+{
+    fileIdsFromFileName(
+        file_name:$file_name, 
+        offset:$offset,
+        first:$first,
+        order_by:$order_by
+    )
+    {
+        file_name
+        file_uuid
+    }
+
+    fileIdsFromFileNameDesc(
```

Comment:
one fileIdsFromFileName should be sufficent

Reply:
updated